src/Entity/BlockManager.php

```diff
@@ -25,42 +24,17 @@
  */
 final class BlockManager extends BaseEntityManager implements BlockManagerInterface
 {
-    public function updatePosition($id, int $position, $parentId = null, $pageId = null, bool $partial = true): PageBlockInterface
+    public function updatePosition($id, int $position): PageBlockInterface
     {
-        $entityManager = $this->getEntityManager();
+        $block = $this->find($id);
 
-        if ($partial) {
```

Comment:
This partial does not make sense to me, there is only one place this method is used, and it always gets false....

@VincentLanglet IMO we should remove it, it even mentions performance problems when is true 😓